### DISCUSSION TITLE
[이지어드민] 배속 속성 타입 코드 확인 문의

### DISCUSSION THREAD

**Discussion by park-junkyu0** [commerce-api-naver/commerce-solutions#18] [묻고 답하기]
안녕하세요, 이지어드민 개발팀 박준규입니다.

**traceId: cr3-161644-647lsv^1724224980926^17878786**
productOrderId: 2024082227426191

해당 주문번호를 당시 조회했을 때, 
productId: 670761692
**deliveryAttributeType: NORMAL**
일반 배송으로 조회가 됐는데요,

고객사에선 해당 상품코드는 21일부터 판매자 도착보장 상품이라는 언급이 있으셔서 확인차 문의드립니다.

호출했을 당시 NORMAL 로 응답값이 표시된 사유 확인 부탁드리겠습니다.

감사합니다.

**Comment by commerce-api-naver** [comment]
@park-junkyu0 

안녕하세요 커머스API 권현철입니다.

말씀하신 내용에 대해 답변 드리겠습니다.

해당 주문 건의 경우 선물하기 주문 건으로 선물하기의 경우 수신인이 배송지를 입력하면 

배송이 시작되므로 도착보장 상품이더라도 일반 배송으로 변경되어져 일반 배송으로 나타나는 것은 정상 현상입니다.

감사합니다.
